linux/Jenkinsfile

```diff
@@ -10,20 +10,20 @@ source      src         -           only for SRPM and no need specify as option
 s390x       s390x       s390x       only for jdk8+
 */
 
-env.NODE_LABEL_RPM = "dockerBuild&&linux&&x64" // RedHat + Suse + Debian x64
-env.NODE_LABEL_DEB = "docker&&linux&&${ARCH}"  // Debian non-x64 
-env.PRODUCT = "temurin"
+env.NODE_LABEL_RPM = 'dockerBuild&&linux&&x64' // RedHat + Suse + Debian x64
```

Comment:
not consistent with single and double quotes?

Reply:
env.NODE_LABEL_DEB has to use double quotes.
but lint wants single quotes for string without variable in it.